mantidproject/mantid
# Add User Support page to dev-docs

**Description of work.**
Mentioned by @LamarMoore 

Adding a User Support page to the dev-docs, which I may add things to over time.

**To test:**
Build dev-docs.html
Test that the link from the dev-docs/source/index.rst file to the UserSupport.rst file works
check the image and format of UserSupport.rst file looks correct

*There is no associated issue.*

*This does not require release notes* because **documentation**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
